### DISCUSSION TITLE
Localization of menu section in nova v4

### DISCUSSION THREAD

**Discussion by pavloniym** [spatie/nova-backup-tool#133] [Q&A]
Hello!
Please help me to understand how can I localize menu section in new nova v4?
In version 3 it was made by `en.json` file in `lang` folder. But now (on v4) there is no effect ...
Thanks!

https://github.com/spatie/nova-backup-tool/blob/79e1beef3167ddacb0f5390078471bea7eec943e/src/BackupTool.php#L25

**Comment by Tam3000** [comment]
In NovaServiceProvider.php

        Nova::mainMenu(function (Request $request) {
            return [
                MenuSection::dashboard(Main::class)->icon('home'),

**Reply by pavloniym** [reply]
I mean how to localize menu section of this backup tool... 
As you can see - name `Backups` is hardcoded, without Laravel default `_(...)` localization tool ...

**Reply by Tam3000** [reply]
Oh sorry. I have the same problem, I'm working on it, when I find the answer I'll let you know.
